test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          setup do
+            @employee = Employee.new(id: 42, name: 'Zoop Zoopler', email: 'zoop@example.com')
+            @picture = @employee.pictures.new(id: 1, title: 'headshot-1.jpg')
+            @picture.imageable = @employee
+
+            @serialization = serializable(@picture, adapter: :json)
+            ActionController::Base.cache_store.clear
```

Comment:
Did you find this cache clear neceassry or it was just copied?  It shouldn't be necessary